argschema/autodoc.py

```diff
@@ -85,11 +85,7 @@ def setup(app):
                     # add this field line to the documentation
                     try:
                         # get the set of types this field was derived from
-                        if isinstance(field, mm.fields.List):
-                            # if it's a list we want to do this for its container
-                            base_types = inspect.getmro(type(field.container))
-                        else:
-                            base_types = inspect.getmro(type(field))
+                        base_types = inspect.getmro(type(field))
```

Comment:
i'm suspicious that this works for all the marshmallow ranges you specified... changing how lists work was one of the things that was moving around a lot..

Reply:
I'll look into it a little more. Maybe I just needed to change `container` to `inner`:
https://github.com/marshmallow-code/marshmallow/blob/65225654ec41ec7528edc7f16d2453d3429cae47/CHANGELOG.rst#300rc8-2019-07-04

Reply:
yes, that was enough. Change limited to just renaming that field.